### PR TITLE
1621 apache ant and maven role

### PR DESCRIPTION
**Pull request by kayiwa**
- add a manual apache tomcat roleA
- add apache_maven role

this is a pulled from my draft 1621 PR to make these more manageable for review.

### CODE REVIEW

File: roles/pulibrary.apache_maven/templates/maven_env.j2 [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Maven environment variables
+# {{ ansible_managed | comment }}
+
+export MAVEN_OPTS="$MAVEN_OPTS -Xmx1024m -XX:PermSize=256m -XX:MaxPermSize=256m -XX:+DisableExplicitGC"
```
Comment (jrgriffiniii) [inline roles/pulibrary.apache_maven/templates/maven_env.j2:6]: 👍

**Comment by jrgriffiniii** [comment]
I am sorry, I am not certain how I closed this in https://github.com/pulibrary/princeton_ansible/pull/1653

**Comment by jrgriffiniii** [comment]
Oh, I see now, a rebase is going to be required after I merged https://github.com/pulibrary/princeton_ansible/pull/1653 - I assume that GitHub now closes PRs when these type of conflicts arise?

In any event, I don't understand why my user account was listed as responsible for closing this when it is an automated process.

**Comment by kayiwa** [comment]
that's a typo on my part.

**Pull request merged**